Concepts overhaul: consolidate basic concepts into 2 page overview, ideas and theory (#1457)
ipfs/ipfs-docs: docs/concepts/ipfs-primer.md

---
title: "Basic Concepts"
description: "Lorem ipsum."
sidebarDepth: 3
---

# What is IPFS?

The InterPlanetary File System (IPFS) is a set of composable, peer-to-peer protocols for addressing, routing, and transferring <VueCustomTooltip label="A way to store information so a device can retrieve the data based on its content, not its location." underlined>content-addressed data</VueCustomTooltip> in a decentralized file system. 

IPFS can be thought of as both the implementations of the IPFS protocols (like <VueCustomTooltip label="The earliest and most widely used implementation of IPFS, written in Go." underlined>Kubo</VueCustomTooltip>) and the decentralized network comprised of the individual <VueCustomTooltip label="Programs that implement the IPFS protocol and participate in the IPFS network." underlined>nodes</VueCustomTooltip> that are running protocol implementations. IPFS _is not_ a <VueCustomTooltip label="A list of records, known as blocks, linked using cryptography. Each block contains a cryptographic hash of the previous block, a timestamp, and transaction data." underlined multiline is-medium>blockchain</VueCustomTooltip>.

> In this conceptual guide, you'll learn about the problems that IPFS addresses, and the subsystems that IPFS uses to address, route, and transfer content-addressed data.

Reviewer: ```suggestion
In this conceptual guide, you'll learn about the problems that IPFS addresses and how the different subsystems that IPFS is comprised of work.
```

I'm not 100% confident about this, but maybe we can keep this broader.
